src/orders/components/OrderDraftDetailsProducts/messages.ts

```diff
@@ -0,0 +1,14 @@
+import { defineMessages } from "react-intl";
+
+export const lineAlertMessages = defineMessages({
+  notPublished: {
+    id: "Oad+ES",
+    defaultMessage: "This product is not published in this channel.",
+    description: "alert message",
+  },
+  notAvailable: {
+    id: "zO+l0L",
+    defaultMessage: "This product is not available for sale in this channel.",
+    description: "alert message",
+  },
```

Comment:
The same message is in `locale/defaultMessages.json`

Reply:
`locale/defaultMessages.json` are auto-generated on purpose for service like Transifex from all ts/tsx files.

Reply:
Ah, alright!